server/node/create_domain.go

```diff
@@ -81,7 +81,7 @@ func (c *CreateDomain) RowIter(ctx *sql.Context, r sql.Row) (sql.RowIter, error)
 		}
 	}
 
-	newType, err := types.NewDomainType(ctx, c.SchemaName, c.Name, c.AsType, defExpr, c.IsNotNull, checkDefs, "")
+	newType, err := types.NewDomainType(c.SchemaName, c.Name, c.AsType, defExpr, c.IsNotNull, checkDefs, "")
```

Comment:
Don't need to change this, but generally better to pass the context and not need it, than to remove the context if it's unused. There have been times where we needed the context later, but didn't have it because we built up a lot of surrounding logic that didn't pass it around.